diff --git a/node_modules/nostr-tools/lib/event.d.ts b/node_modules/nostr-tools/lib/event.d.ts
index bf2d50f..5bf0a06 100644
--- a/node_modules/nostr-tools/lib/event.d.ts
+++ b/node_modules/nostr-tools/lib/event.d.ts
@@ -22,7 +22,8 @@ export declare enum Kind {
     HttpAuth = 27235,
     ProfileBadge = 30008,
     BadgeDefinition = 30009,
-    Article = 30023
+    Article = 30023,
+    FileMetadata = 1063
 }
 export type EventTemplate<K extends number = Kind> = {
     kind: K;
